pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ReaderConfigurationData.java

```diff
@@ -32,6 +32,7 @@
 
     private String topicName;
     private MessageId startMessageId;
+    private Long timestamp = -1L;
```

Comment:
call it `publishTime` or `publishTimestamp`. because `timestamp` is an ambiguous term, it has multiple meanings in pulsar - publish time or event time.